docs/dxp/7.x/en/site-building/05-personalizing-site-experience/01-segmentation/assigning-roles-to-user-segments.md

```diff
@@ -1,6 +1,6 @@
 # Assigning Roles to User Segments
 
-Since Liferay Portal CE 7.3 SP2, you can assign User Segments to Roles. Follow these steps:
+Since Liferay Portal CE 7.3 GA1, you can assign User Segments to Roles <!-- Should this be: "you can assign roles to user segments"? This task enables users to make it so anyone who has role [x] is part of segment [y], is that right? -->. Follow these steps:
 
```

Comment:
Yes, this is assigning Roles to a User Segment, but it doesn't automatically assign someone who has that role to the User Segment.